tox.ini

```diff
@@ -44,9 +44,7 @@ commands =
     python --version
     pip --version
     python ci/install_wheels.py
-    python -c "from gensim.models.word2vec import FAST_VERSION; print(FAST_VERSION)"
```

Comment:
Need to add explicit cython compilation in `setup.py`

Reply:
How do we do this? I had a look through http://docs.cython.org/en/latest/src/quickstart/build.html, it looks like we're already including `ext_modules`. What else can we do?

Reply:
1. Add `cython` to install requirements
2. Remove "python fallback"
3. Cleanup all `.so` from folder
4. Run `python setup.py install` and make sure than all `.so` generated automatically, if not - ping me, I'll have a look myself